CMake/external_fastdds.cmake

```diff
@@ -15,8 +15,9 @@ function(get_fastdds)
 
     FetchContent_Declare(
       fastdds
-      GIT_REPOSITORY https://github.com/eProsima/Fast-DDS.git
-      GIT_TAG        7124ff87c4813bed26858fd89d3b915a5857dbc7 # Git tag "v2.6.0", when updating this version consider removing the patches listed below.
+      # TODO Eran's repo has a fix for #3131
```

Comment:
It's temporary... we're actively working on this. We have an open issue with them, I'm sure it'll get fixed.

Reply:
I prefer to take the newer version once fixed and not go away from the main branch.
I see no benefit here. let's talk.

Reply:
This is a branch that's WIP and not public. What's the issue?

Reply:
Too much to handle.
We cannot cover each PR with a temporary solution, since we are not blocked with version 2.6.0 and needs 2.8.2 then I think we can wait and update the proper way.
Tomorrow if we will have other priorities and we are left with few resources for DDS it will remain on your fork for years.
Patches are for a must solution, otherwise I prefer to go with the traditional way.
We are making too many changes at once.

Reply:
Alright, I'll remove the 2.8.1 move and just leave the CI